local/compose/push.go

```diff
@@ -70,12 +70,7 @@ func (s *composeService) Push(ctx context.Context, project *types.Project, optio
 				if !options.IgnoreFailures {
 					return err
 				}
-				w.Event(progress.Event{
-					ID:         fmt.Sprintf("Pushing %s:", service.Name),
-					Text:       fmt.Sprintf("%v", err),
-					Status:     progress.Error,
-					StatusText: fmt.Sprintf("%s", err),
-				})
+				w.TailMsgf("Pushing %s: %s", service.Name, err.Error())
```

Comment:
Doing so, we will show the error as tail message 👍  but the progress won't complete. Shouldn't we do both?

Reply:
From what I see this Error event is unique. It has no `progress.Working` preceding it.